allure-pytest/test/acceptance/parametrization/parametrization_test.py

```diff
@@ -173,3 +173,21 @@ def test_dynamic_parameter_override_from_fixture(executed_docstring_source):
                               has_parameter("param1", "'readable-value'")
                               )
                 )
+
+
+def test_fullname_with_braces(executed_docstring_source):
+    """
+    >>> import pytest
+    ... import allure
+
+    >>> class TestClass:
+    ...     @pytest.mark.parametrize("param1", ["qwe]["])
+    ...     def test_with_braces(self, param1):
+    ...         pass
+    """
+    assert_that(executed_docstring_source.allure_report,
+                has_test_case("test_with_braces[qwe][]",
+                              has_entry('fullName', ends_with("test_with_braces")),
```

Comment:
Probably we should modify the assertion a little to catch the abovementioned dot absent:
```suggestion
                              has_entry('fullName', ends_with(".TestClass#test_with_braces")),
```

Reply:
nice catch! thank you